app/scripts/services/api-explorer.js

```diff
@@ -115,36 +137,37 @@
                                         apiGroup = apiGroupTags[0].name;
                                     }
                                     angular.forEach(filterFilter(value.tags, {category: "product"}, true), function(value, index) {
-                                    	products.push(value.name);
+                                        products.push(value.name);
                                     });
 
                                     angular.forEach(filterFilter(value.tags, {category: "programming-language"}, true), function(value, index) {
-                                    	languages.push(value.name);
+                                        languages.push(value.name);
                                     });
                                 }
                             }
 
                             // Clean the type
                             if (type == "iframe-documentation" || (value.api_ref_doc_url && value.api_ref_doc_url.endsWith(".html"))) {
-                           		type = "html";
+                                   type = "html";
```

Comment:
It is not necessarily a bad thing that you reformatted this entire file since it is a good thing to have the files be consistent in formatting.  That said, since nearly every line in the file is modified, it makes reviewing really laborious.  Please don't couple together massive reformatting with feature additions in the future.